azurerm/resource_arm_stream_analytics_reference_input_blob_test.go

```diff
@@ -0,0 +1,394 @@
+package azurerm
```

Comment:
The tests in here are also pretty much a mirror of the "stream"_input_blob resource's tests.